README.md

```diff
@@ -1 +1,17 @@
-# hello-microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> A basic HTML & CSS project built with Lighthouse, WebHint and Stylelint for the purpose of understanding these tools and get familiar with them at the later projects.
+
+## Built With
+
+- HTML, CSS, Node
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@tugsem]
+- Twitter: [@TugsemSaranli]
+- LinkedIn: [Tugsem Saranli]
```

Comment:
- [ ] Kindly add the links to your accounts. So users can reach you.
like 
- GitHub: `[tugsem](HTTP://................)`